snuba/clickhouse/columns.py

```diff
@@ -302,3 +304,23 @@ def get(self, key, default=None):
 
     def for_schema(self):
         return ', '.join(column.for_schema() for column in self.columns)
+
+
+class QualifiedColumnSet(ColumnSet):
```

Comment:
This preserves some structure in the ColumnSet when we have columns coming from different tables.
It still allows iteration the way we did before (which is what is needed in util.conditions_expr) but it does not loose the which column comes from which table as the previous implementation did.